addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -556,10 +556,13 @@ var LineRenderer = Widget.extend(FieldManagerMixin, {
         }, {
             type: 'float',
             name: 'amount',
+        }, {
+            type: 'char', //TODO is it a bug or a feature when type date exists ?
```

Comment:
this might works as we instanciate the fields below but it would be better to change to type: 'date'

Reply:
Yeah, I went to the JS framework team for 3h and they couldn't help me make the 'date' type work

Reply:
if they could not make it works, let's leave it like this